origin-dapp/src/components/message.js

```diff
@@ -25,14 +25,48 @@ class Message extends Component {
       message,
       messagingEnabled,
       user,
-      contentOnly
+      contentOnly,
+      mobileDevice
     } = this.props
     const { created, hash } = message
     const { address, fullName, profile } = user
 
     return contentOnly ? (
       <div className="d-flex compact-message">{this.renderContent()}</div>
-    ) : (
+    ) : mobileDevice ? (
```

Comment:
What do you think about having several conditioned `return` statements instead of chaining ternaries together? I could go either way but it's a bit difficult to read as-is.

Reply:
I agree it is difficult to read, I'll update